controllers/employeeController.js

```diff
@@ -108,3 +110,20 @@ export const employeeSuspend = async (req, res) => {
     });
   }
 };
+
+export const employeeSearch = async (req, res) => {
+  try {
+    const [keys] = Object.keys(req.body);
+    const [values] = Object.values(req.body);
+    const result = await Employees.find({
+      [keys]: { $regex: values },
```

Comment:
can you please fix the issue! I was not able to search by phone number before merge.
An alternative you can try is: 
```
      - Find all employees with no condition
      - Filter the employees before sending the response
```

Reply:
you can also provide a number of found results as an addition

Reply:
provided a number of found matches but searching by phone is not yet done

Reply:
awesome job keep looking for another way to search for numbers. I will try to provide you with some help

Reply:
done but faked